tofu/geom/_GG.pyx

```diff
@@ -2982,19 +2982,19 @@ def LOS_calc_signal(func, double[:,::1] ray_orig, double[:,::1] ray_vdir, res,
                                    num_threads)
             # ..................................................................
         if ani:
-            val_2d = func(pts, t=t, vect=-usbis, **fkwdargs)
+            val_arr = func(pts, t=t, vect=-usbis, **fkwdargs)
```

Comment:
val_arr does not seem to have been declared, is it voluntary ?

Reply:
we declare for speed ups, see my next comment. (Sorry some times I have to push in order to try it in the ITER clusters).